code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1980,15 +1980,30 @@ mob/living/carbon/human/isincrit()
 	else
 		return ..()
 
+/mob/living
+	var/hangman_score = 0 // For round end leaderboards
+
 /mob/living/carbon/human/Hear(var/datum/speech/speech, var/rendered_speech="")
 	..()
 	if(stat)
 		return //Don't bother if we're dead or unconscious
 	if(ear_deaf || speech.frequency || speech.speaker == src)
 		return //First, eliminate radio chatter, speech from us, or wearing earmuffs/deafened
+	var/mob/living/H = speech.speaker
+	if(muted_letters && muted_letters.len && length(speech.message) == 1)
+		if(speech.message in muteletters_check)
+			muted_letters.Remove(speech.message)
+			H.visible_message("<span class='notice'>[speech.speaker] guessed a correct letter!</span>","<span class='notice'>You guessed a correct letter!</span>")
```

Comment:
This feels like the kind of feedback I'd receive from an official hangman PDA app. It seems... weird to have this displayed from a human, presumably by magical forces. I might go with more magical-seeming span classes and make the messages themselves more enchanted sounding.

Reply:
Changed it